Add authorized name

p-galligan opened the pull request:
Adds authorized name property to the agent schema.

I have not attempted any regex to enforce indirect order because that would not be upheld for the corporate agent types. We could do some if/then statements that would add complexity here that probably isn't necessary.

p-galligan committed:
add authorized name

Add the authorized name property to agent definition.

p-galligan committed:
tweak agent order

Tweak order of required properties in agents.

helrond commented:
I don't think we need to try and enforce inverted order for people - I'm not sure how we would do that other than a regex which looks for a comma, which would be potentially inaccurate for people who have a comma in the direct form of name (like Jrs, etc).

p-galligan committed:
Update tests

Update valid tests to include authorized_name. Added field to invalid tests testing other fields.

helrond reviewed (approved):
Nice, looks good!

helrond commented:
When you merge this, remember to cut a new release.
